api-reference/10 UI Components/GridBase/1 Configuration/filterBuilder.md

```diff
@@ -8,11 +8,18 @@ default: {}
 Configures the integrated filter builder.
 
 ---
-See the [FilterBuilder configuration](/api-reference/10%20UI%20Components/dxFilterBuilder/1%20Configuration '/Documentation/ApiReference/UI_Components/dxFilterBuilder/Configuration/') for properties that you can specify in this object.
+See the [FilterBuilder configuration](/api-reference/10%20UI%20Components/dxFilterBuilder/1%20Configuration '/Documentation/ApiReference/UI_Components/dxFilterBuilder/Configuration/') for properties that you can specify in this object. Do not specify the [fields](/Documentation/ApiReference/UI_Components/dxFilterBuilder/Configuration/fields/) array because the {WidgetName} automatically populates it to keep filter builder fields in sync with grid columns. 
```

Comment:
```suggestion
See the [FilterBuilder configuration](/api-reference/10%20UI%20Components/dxFilterBuilder/1%20Configuration '/Documentation/ApiReference/UI_Components/dxFilterBuilder/Configuration/') for properties that you can specify in this object. Do not specify the [fields](/Documentation/ApiReference/UI_Components/dxFilterBuilder/Configuration/fields/) array because the {WidgetName} automatically populates it to sync filter builder fields with grid columns. 
```